README.rst

```diff
@@ -52,7 +45,23 @@ This is optional. You can set your own return url by adding this to `settings.py
 
     PESAPAL_OAUTH_CALLBACK_URL = 'app_name:url_name'  # this needs to be a reversible
 
-#. Run `python manage.py syncdb` to create the polls models.
+The url can have the 'merchant_reference' passed to it as a parameter for further processing.
+Or be without a parameter.
+
+for example the url pattern
+
+r'^response/$'
+
+without the merchant reference parameter
+
+or
+
+r'^response/(?P<merchant_reference>\d+)/$'
```

Comment:
I think making the query string take the following pattern  (?P<merchant_reference>.*?) would be best, since the same pattern is used for slugs meaning that the parameter could take the form of an integer/uuid or string.